app/views/homeworks/_problem_fields.html.erb

```diff
@@ -1,5 +1,6 @@
 <div class='nested-fields'>
   <%= f.input :name, label: "Имя (опционально)" %>
-  <%= f.input :text, label: "Условие", input_html: { rows: 3 } %>
+  <%= f.input :text, label: "Условие", input_html: { rows: 3 },
+      hint: "Нажмите Ctrl+Enter, чтобы отправить задачу, Enter или Shift+Enter для переноса строки"%>
```

Comment:
So, you think it is a good idea to use `Enter` in a chat-like way on a big text field?